src/domain/Prize.js

```diff
@@ -0,0 +1,45 @@
+export const FIRST_PRIZE = 2000000000;
+export const SECOND_PRIZE = 30000000;
+export const THIRD_PRIZE = 1500000;
+export const FOURTH_PRIZE = 50000;
+export const FIFTH_PRIZE = 5000;
+
+export const PRIZE = {
+  FIRST: { rank: "FIRST", matchCount: 6, prize: FIRST_PRIZE, isBonus: false },
+  SECOND: {
+    rank: "SECOND",
+    matchCount: 5,
+    prize: SECOND_PRIZE,
+    isBonus: true,
+  },
+  THIRD: { rank: "THIRD", matchCount: 5, prize: THIRD_PRIZE, isBonus: false },
+  FOURTH: {
+    rank: "FOURTH",
+    matchCount: 4,
+    prize: FOURTH_PRIZE,
+    isBonus: false,
+  },
+  FIFTH: { rank: "FIFTH", matchCount: 3, prize: FIFTH_PRIZE, isBonus: false },
+  NONE: { rank: "NONE", matchCount: 0, prize: 0, isBonus: false },
+};
+
+class Prize {
+  #rank = [PRIZE.FIRST, PRIZE.SECOND, PRIZE.THIRD, PRIZE.FOURTH, PRIZE.FIFTH];
```

Comment:
```suggestion
class Prize {
  RANK = {
      FIRST: { rank: "FIRST", matchCount: 6, prize: FIRST_PRIZE, isBonus: false },
      SECOND: {
        rank: "SECOND",
        matchCount: 5,
        prize: SECOND_PRIZE,
        isBonus: true,
      },
      THIRD: { rank: "THIRD", matchCount: 5, prize: THIRD_PRIZE, isBonus: false },
      FOURTH: {
        rank: "FOURTH",
        matchCount: 4,
        prize: FOURTH_PRIZE,
        isBonus: false,
      },
      FIFTH: { rank: "FIFTH", matchCount: 3, prize: FIFTH_PRIZE, isBonus: false },
      NONE: { rank: "NONE", matchCount: 0, prize: 0, isBonus: false },
    };
```
이렇게 아예 class 내에 상수로 선언하는 방법도 있답니다!

Reply:
밖에서도 RANK를 사용하기 때문에 export하여 사용하고 있었습니다! 음..class 내에 상수로 선언해서 함수로 분리하는게 좋을까요?